### PR TITLE
Added docker contanerization to png-extraction module as a reponse to #262

### PR DESCRIPTION
**Pull request by Nitesh639**
As a response to #262 . Added steps to follow in README.md  
Has been tested several times by several people.

### CODE REVIEW

**Comment by pradeeban** [comment]
Thanks. Please always send pull requests against the dev branch.

I will shortly add more comments in the commit itself. Please address those as well.

File: modules/png-extraction/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,12 @@
+FROM ravirahar/arch-python-gdcm
```
Comment (pradeeban) [inline modules/png-extraction/Dockerfile:1]: This should come from a standard base image (such as Ubuntu, Python, ...). Not from your personal arch-python-gdcm base.

File: modules/png-extraction/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+ARG DICOMHome="/opt/data/new-study"
```
Comment (pradeeban) [inline modules/png-extraction/Dockerfile:2]: This should take from a config.json value that is provided by the user, copied from the host to the container. Not a hard-coded value.

Comment (Nitesh639) [reply modules/png-extraction/Dockerfile:2]: It is just the default value and not hard coded value. User is not to touch Dockerfile instead use png-extraction-docker script which has options to set both DICOMHome and OutputDirectory.

File: modules/png-extraction/Dockerfile (hunk 1 above)
Comment (Nitesh639) [reply modules/png-extraction/Dockerfile:1]: Okay, I will make the changes and make pull request again. It was just that I wanted to use lightweight linux and some modules on arch, like gdcm were only on AUR. But as you say, I will use some standard image.

**Pull request closed without merging**